alot/commands/envelope.py

```diff
@@ -205,6 +232,16 @@ def __init__(self, mail=None, envelope=None, **kwargs):
                                     msg_position='left')) == 'no':
                     return
 
+            # Check if an encrypted message is indeed encrypted to all
+            # recipients.
+            if self.envelope.encrypt and not self._check_recipient_keys():
+                warning = textwrap.dedent("""\
+                    Message is not encrypted to all recipients. Do you want to
```

Comment:
can we extend this message to include the consequences?
"This means that not everyone will be able to decode and read this message"
or "This means that the message will be sent out in plain text as well". I can't remember the semantics.